doc/md/stable-memory/stablememory.md

```diff
@@ -2,102 +2,20 @@
-
-
-``` motoko no-repl
-module {
-
-  // Current size of the stable memory, in pages.
-  // Each page is 64KiB (65536 bytes).
-  // Initially `0`.
-  size : () -> (pages : Nat64);
-
-  // Grow current `size` of stable memory by `pagecount` pages.
-  // Each page is 64KiB (65536 bytes).
-  // Returns previous `size` when able to grow.
-  // Returns `0xFFFF_FFFF_FFFF_FFFF` if remaining pages insufficient.
-  grow : (new_pages : Nat64) -> (oldpages : Nat64);
-
-  loadNat8 : (offset : Nat64) -> Nat8;
-  storeNat8 : (offset : Nat64, value: Nat8) -> ();
-
-  // ... and similar for Nat16, Nat32, Nat64,
-  // Int8, Int16, Int32 and Int64 ...
-
-  loadFloat : (offset : Nat64) -> Float;
-  storeFloat : (offset : Nat64, value : Float) -> ();
-
-  // Load `size` bytes starting from `offset` as a [`Blob`](../base/Blob.md).
-  // Traps on out-of-bounds access.
-  loadBlob : (offset : Nat64, size : Nat) -> Blob;
-
-  // Write bytes of [`Blob`](../base/Blob.md) beginning at `offset`.
-  // Traps on out-of-bounds access.
-  storeBlob : (offset : Nat64, value : Blob) -> ()
-
-  // Returns a query that, when called, returns the number of bytes of
-  // (real) IC stable memory that would be occupied by persisting its
-  // current stable variables before an upgrade.
-  stableVarQuery : () -> (shared query () -> async {size : Nat64})
-}
-```
-
-## Example
-
-To demonstrate the `ExperimentalStableMemory` library, we present a dead simple implementation of a logging actor that records text messages in a scalable, persistent log.
-
-The example illustrates the simultaneous use of stable variables and stable memory. It uses a single stable variable to keep track of the next available offset, but stores the contents of the log directly in stable memory.
-
-``` motoko no-repl file=../examples/StableLog.mo
-```
-
-The auxiliary function `ensure(offset)` is used to grow `ExerimentalStableMemory` as necessary to accommodate more data. It computes the 64KiB page of a given offset and ensures enough pages have been allocated to guarantee that offset is within bounds.
-
-The shared `log(t)` function encodes its [`Text`](../base/Text.md) argument as a [`Blob`](../base/Blob.md), allocates enough stable memory to store it, and writes both the blob contents and its size at the next available offset in `ExperimentalStableMemory`, updating `base`.
-
-The shared `readLast(count)` query reads up to `count` messages from the log, traversing the log in reverse from `base`.
-
-Because `StableLog` allocates and maintains its (potentially large) log data directly in stable memory and uses just a small and fixed amount of storage for actual stable variables (here `base`), upgrading `StableLog` to a new implementation (perhaps to provide more functionality) should not consume too many cycles, regardless of the current size of the log.
-
```

Comment:
I am not sure whether we should remove the packages mentioned below too, especially if they only operate on ESM and not on regions. I did not yet check this.